crates/rolldown/tests/esbuild/default/re_export_default_external_es6/artifacts.snap

```diff
@@ -0,0 +1,29 @@
+---
+source: crates/rolldown/tests/common/case.rs
+expression: content
+input_file: crates/rolldown/tests/esbuild/default/re_export_default_external_es6
+---
+# warnings
+
+## UNRESOLVED_IMPORT
+
+```text
+[UNRESOLVED_IMPORT] Warning: "foo" is imported by "entry.js", but could not be resolved – treating it as an external dependency.
+
+```
+## UNRESOLVED_IMPORT
+
+```text
+[UNRESOLVED_IMPORT] Warning: "bar" is imported by "bar.js", but could not be resolved – treating it as an external dependency.
+
+```
+# Assets
+
+## entry_js.mjs
+
+```js
+import { default as foo } from "foo";
+import { default as bar } from "bar";
+
+export { bar, foo };
+```
```

Comment:
Could you solve the `unresolved` error? esbuild snapshot does not have these errors.

Reply:
Ok, will be resolved tomorrow